site/src/components/BuildAvatar/BuildAvatar.tsx

```diff
@@ -17,7 +17,7 @@ export const BuildAvatar: FC<BuildAvatarProps> = ({ build, size }) => {
 	const theme = useTheme();
 	const { status, type } = getDisplayWorkspaceBuildStatus(theme, build);
 	const badgeType = useClassName(
-		(css, theme) => css({ backgroundColor: theme.palette[type].light }),
+		(css, theme) => css({ backgroundColor: theme.roles[type].fill.solid }),
```

Comment:
Something I've kept forgetting to ask about: what is the difference between `palette` and `roles` again? Is one newer than the other, or are they meant to address different concerns?

Reply:
`roles` is supposed to replace `palette`. I'll add a deprecation note to `palette`. :)